.github/workflows/build.yml

```diff
@@ -49,7 +49,8 @@ jobs:
       run: npm run lint
     - name: npm run build
       working-directory: ./client/webserver/site
-      run: npm run build
+      run: ./ci_tests.sh
```

Comment:
Well, hmm it could get long that way.  Whatever I guess, but maybe the script should be `check_cache_busters.sh` and `npm run build` can remain outside of it (or include both `npm ci` and `npm run build` too in the script)